keyboards/stenokeyboards/polyglot/keymaps/everything_qwerty/keymap.c

```diff
@@ -0,0 +1,218 @@
+      XXXXXXX,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                   KC_1,   KC_C,    KC_V,                        KC_N,    KC_M, KC_2
+                                      //`--------------------------'  `--------------------------'
+
+  ),
+
+  [_GEMINI] = LAYOUT(
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+      BACK,  STN_N1,  STN_N2,  STN_N3,  STN_N4,  STN_N5,  					            STN_N6,  STN_N7,  STN_N8,  STN_N9,  STN_NA,  STN_NB,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX, STN_S1,  STN_TL,  STN_PL,  STN_HL,  STN_ST1, 					             STN_ST3, STN_FR,  STN_PR,  STN_LR,  STN_TR,  STN_DR,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX, STN_S2,  STN_KL,  STN_WL,  STN_RL,  STN_ST2, 					             STN_ST4, STN_RR,  STN_BR,  STN_GR,  STN_SR,  STN_ZR,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                       STN_N1,   STN_A,   STN_O,         STN_E,   STN_U, STN_N2
+                                      //`--------------------------'  `--------------------------'
+
+  ),
+
+  [_UNI_PLOVER] = LAYOUT(
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+      XXXXXXX,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_LBRC,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+        BACK,  KC_LCTL,  KC_LGUI,  KC_LALT,  KC_LSFT,  KC_SPC,    				KC_VOLD,  KC_LEFT, KC_DOWN, KC_UP ,KC_RIGHT,  KC_VOLU,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                    KC_1,   KC_C,    KC_V,                        KC_N,    KC_M, KC_2
+                                      //`--------------------------'  `--------------------------'
+
+  ),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record)
+{
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
```

Comment:
it looks like you want to do the default layer functions/keycodes, rather than this. 

https://docs.qmk.fm/#/feature_layers?id=functions

Reply:
I don't know...

I need to have the Uni Gemini layer to be the default starting layer.

The "back" key changes the layer to a "default" qwerty/Dvorak/Colemak layer. It's not really a default layer, because the default layer is the Uni Gemini layer.

Reply:
That still sounds like default layers to me.  For reference: 
https://docs.qmk.fm/#/keymap?id=keymap-and-layers

Note that these aren't persistent, by default.